packages/compiler/src/core/numeric.ts

```diff
@@ -145,10 +152,11 @@ const compare = (a: InternalData, b: InternalData): 0 | 1 | -1 => {
   } else if (a.s > b.s) {
     return 1;
   }
+  const neg = a.s;
   if (a.e < b.e) {
-    return -1;
+    return (-1 * neg) as any;
```

Comment:
Is it worth having this so the return type of compare is `0 | 1 | -1` or should just update to number

Reply:
I think the current signature is cute, and marginally helpful. `any` is fine.